membership.md

```diff
@@ -4,19 +4,16 @@ title: członkostwo
 permalink: /membership/
 menus: header
 ---
-Aby zostać członkiem musisz opłacic składki w całości zużywane na utrzymaie HS. [Jako organizacja non profit opieramy się na wsparciu lokalnej społeczności.](https://patronite.pl/hs3) Potrzebujemy pieniędzy na pokrycie stałych kosztów wynajmu lokalu. Wszelki nadmiar gotówki zostaje przeznaczony na zebranie pieniężnej trzymiesięcznej "żelaznej racji" oraz zakup sprzętu. Jeśli dysponujesz dodatkowym groszem lub masz u siebie, w firmie lub u znajomych sprzęt na zdanie, to daj nam znać.
 
-Po opłaceniu składki zapraszamy na kilka spotkań byśmy się poznali. Kiedy dostaniesz polecnie dwóch członków rady HSu i jednego członka zwykłego dostaniesz klucz do lokalu. Będziesz wtedy przez 3 msc na okresie próbnym. Po czym rada przeprowadzi głosowanie czy przyznać Ci członkostwo.
+Przede wszystkim wpadnij na jakieś interesujące Cię wydarzenie i poznaj ludzi (/calendar). W piątki mamy zazwyczaj drzwi otwarte - to świetna okazja, żeby poznać członków i dowiedzieć się czym jest HS. W razie czego można też przyjść w inne dni, ale wtedy skontaktuj się z nami wcześniej, bo nie wiadomo czy ktoś będzie akurat siedział (/contact).
 
-Nicki członków rady na slacku: @amadeusz, @laracroft, @marcin, @jogurt, @rrr, @patwys, @not7cd, @allgreed
+Członkostwo daje Ci dostęp do lokalu (własna karta do drzwi) oraz większe możliwości współtworzenia spejsu. Zdobycie członkostwa wymaga przede wszystkim poznania członków i zdobycia 3 okejek. Okejki są sygnałem, że ktoś za Ciebie poręczył. Drugi wymóg to pomaganie HSowi w utrzymaniu lokalu poprzez wpłacanie 100 PLN/msc. przez minimum 3 miesiące.
```

Comment:
Minimum stówkę miesięcznie, można więcej :D

Reply:
coś w stylu:
"poprzez wpłacanie 100 PLN/msc. (lub więcej) przez minimum 3 miesiące."
albo:
"poprzez wpłacanie minumum 100 PLN/msc. przez 3 miesiące."

nie chcę pisać dwa razy minimum obok siebie.

Genralnie nie wiem czy kwestia tego, że potem dalej trzeba płacić jest jasna :P
Na początku pisałem że to składka, ale potem shitsailor zwrócił mi uwagę, że to mało intuicyjne, że opłaca się składkę ale nie jest się członkiem.

Reply:
Myślę, że jest ok. Tutaj chcemy info w pigułce, o tym że można płacić więcej możemy mówić na spotkaniach.